parlacards/scores/attendance.py

```diff
@@ -83,7 +80,10 @@ def calculate_group_vote_attendance(group, timestamp=datetime.now()):
 
         ballots = ballots.union(member_ballots.filter(q_ballot_objects))
 
-    ballot_options = ballots.values_list('option', flat=True)
+    # WORKAROUND because values_list of union returns distinct values
+    #ballot_options = ballots.values_list('option', flat=True)
+    ballot_options = [ballot.option for ballot in list(ballots)]
```

Comment:
tu verjetno ne rabiš `in list(ballots)` ampak lahko samo `in ballots`

Reply:
Rabiš. Ker se očitno neki izvajal async in pol vrže exeeption če se hočeš iterirat po tistem QuerySetu